src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable consistent-return */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `/* eslint-disable consistent-return */` at the top of your file disables a useful ESLint rule that ensures all code paths in a function return a value. This can help prevent bugs related to undefined behavior. If there's a specific function where you believe this rule should not apply, consider disabling it only for that function.

```diff
- /* eslint-disable consistent-return */
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->